src/components/Tooltip/Tooltip.jsx

```diff
@@ -129,12 +130,16 @@ Tooltip.propTypes = {
 
   /** The tooltip position */
   hotkey: PropTypes.string,
+
+  /** Custom HTML */
+  customHTML: PropTypes.shape({ __html: PropTypes.string }),
```

Comment:
If this is expected to be a jsx element, it might be worth using the `PropTypes.node` instead of `shape` here. Then it will support text entities, and more. Or you could use `PropTypes.element` if you explicitly want an element. Or perhaps a union type `PropTypes.oneOf([PropTypes.element, null])`. Theres more info to help you make that choice in the proptypes documentation https://reactjs.org/docs/typechecking-with-proptypes.html